### ISSUE TITLE
Menubar: Aria label not set to item label

### ISSUE THREAD

**Issue by lisuzanne94** [primefaces/primevue#5723]
### Describe the bug

The aria-labels for the menubar items should be set to the item's label. Instead, they are all being set to `aria-label="1"`.
```
<script setup>
import Menubar from 'primevue/menubar'
import { ref } from 'vue'

const items = ref([
    {
      label: 'label-one',
      items: [
        {
          label: 'sub-one',
        }
      ]
    },
    {
      label: 'label-two',
      items: [
        {
          label: 'sub-two-one',
        },
        {
          label: 'sub-two-two',
        },
        {
          label: 'sub-two-three',
          items: [
            {
              label: 'sub-sub-two-one',
            },
            {
              label: 'sub-sub-two-two',
            },
          ]
        }
      ]
    },
    {
      label: 'label-three',
      items: [
        {
          label: 'sub-three',
        }
      ]
    },
    ])
</script>

<template>
  <Menubar :model="items" />
</template>

<style scoped>
</style>
```
![Screenshot 2024-05-10 at 1 10 44 PM](https://github.com/primefaces/primevue/assets/80358332/654db7eb-66ff-4208-a442-d24d1097313a)

### Reproducer

Wasn't able to reproduce in sandbox

### PrimeVue version

^3.52.0

### Vue version

3.x

### Language

ES6

### Build / Runtime

Vite

### Expected behavior

The menubar `li` elements should have `aria-label="label-one"`, `aria-label="label-two"`, etc. But they all have `aria-label="1"` instead.

**Comment by lisuzanne94** [comment]
Found the issue. We were using AudioEye which was interfering with the aria labels.
